.github/workflows/release_main.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       uses: docker/build-push-action@7f9d37fa544684fb73bfe4835ed7214c255ce02b
       with:
         push: true
-        tags: stellar/anchor-platform:${{ github.event.release.tag_name }},stellar/anchor-platform:latest
+        tags: stellar/anchor-platform:${{ github.event.release.tag_name }}
```

Comment:
Isn't this disabling release to latest for all branches?

Reply:
This only modifies the `legacy/main-1.x` workflow https://github.com/stellar/java-stellar-anchor-sdk/actions/runs/5693240869/workflow. This is different from the workflow publishing releases from develop https://github.com/stellar/java-stellar-anchor-sdk/actions/workflows/wf_release_created_or_updated.yml.

Reply:
Ok, lets fix the tests and then I'll approve

Reply:
The tests are dependent on the reference server preview which seem to be broken on the Kubernetes side. Since this isn't affecting functionality, are you ok with approving as is?

Reply:
@philipliu Are you suggesting to have another PR to fix the kube?